bundles/org.openhab.binding.insteon/src/main/java/org/openhab/binding/insteon/internal/device/CommandHandler.java

```diff
@@ -346,6 +346,7 @@ public void handleCommand(InsteonChannelConfiguration conf, Command cmd, Insteon
                             getGroup(conf));
                     Msg m = dev.makeStandardMessage((byte) 0x0f, cmd1, value, group);
                     dev.enqueueMessage(m, feature);
+                    feature.pollRelatedDevices();
```

Comment:
How do you know that polling will occur after the enqueued message?

Reply:
This bascially replicates what happens in DeviceFeature.publishState() -> DeviceFeatureListener.publishState() when a new state comes in from Insteon network. Since this is a broadcast message, and the way the Insteon protocol works state changes are not sent to the broadcaster.

Reply:
I've also tested to make sure the polling is occurring as expected.

Reply:
@mfedotov has tested as well.